src/main/java/com/baidu/hugegraph/formatter/KGDumper/ComputeSign.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2017 HugeGraph Authors
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with this
+ * work for additional information regarding copyright ownership. The ASF
+ * licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.baidu.hugegraph.formatter.kgdumper;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.LinkedHashMap;
+
+public class ComputeSign {
+
+    private Map<String, Integer> tradeValue2Num;
+    private LinkedHashMap<String, String> entPlain2Id;
+
+    private String inputEncoding;
+
+    public ComputeSign(String tradeValueFile, String inputEncode) throws IOException {
+        File file = new File(tradeValueFile);
+        Long fileLen = file.length();
+        byte[] fileBytes = new byte[fileLen.intValue()];
+        FileInputStream in = null;
+        try {
+            in = new FileInputStream(file);
+            in.read(fileBytes);
+            in.close();
```

Comment:
remove "in.close();" since finally-block will do it.

or we can just use auto close:
```
try (FileInputStream in = new FileInputStream(file)) {
      in.read(fileBytes);
} // unneeded to close and finally
```